src/components/home-banner/home-banner.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+
+import { StarIcon } from "../star-icon/star-icon";
+import { PolygonIcon } from "../polygon-icon/polygon-icon";
+
```

Comment:
Maybe you can define an icon folder and put these components into this folder

Reply:
Done